app/src/main/kotlin/com/wire/android/ui/home/conversations/details/menu/ConversationGroupDetailsBottomSheet.kt

```diff
@@ -17,12 +17,13 @@ import com.wire.android.ui.common.conversationColor
 import com.wire.android.ui.common.dimensions
 import com.wire.android.ui.home.conversations.details.options.GroupConversationOptionsState
 import com.wire.android.ui.home.conversationslist.common.GroupConversationAvatar
+import com.wire.android.ui.home.conversationslist.model.LeaveGroupState
 
 @Composable
 fun ConversationGroupDetailsBottomSheet(
     conversationOptionsState: GroupConversationOptionsState,
-    onDeleteGroup: () -> Unit,
-    onLeaveGroup: () -> Unit,
+    onDeleteGroup: (Unit) -> Unit,
```

Comment:
What's the point of declaring Unit as the input parameter instead of nothing?

Reply:
It will be changed after adding also state to delete dialog